src/Components/Onboarding/Steps/CollectorIntent.tsx

```diff
@@ -13,6 +14,10 @@ const OptionsContainer = styled.div`
   &:last-child {
     border-bottom: 1px solid #e5e5e5;
   }
+  ${media.sm`
+    width: 335px;
```

Comment:
Does a fixed width work well on different sizes of mobile screens?

Reply:
I used the fix width of the value in the spec but I can make this a % value. I was noticing that we only have specs for mobile and not ipad devices so that's another consideration.